packages/react/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -0,0 +1,44 @@
+import { IComboBoxProps } from '../../ComboBox';
+
+// Can only be numbers between 0-23
+export interface ITimeRange {
+  start: number;
+  end: number;
+}
```

Comment:
Rather than using a `//` comment here, you should use `/** doc comments */` so that they'll show up in intellisense. It's also good to be more explicit about what the props do.

Also, I'm guessing `end` is exclusive? But if not please update the wording.
```suggestion
/**
 * Range of start and end hours to be shown in the TimePicker.
 */
export interface ITimeRange {
  /** Start hour (inclusive) for the time range, 0-23 */
  start: number;
  /** End hour (exclusive) for the time range, 0-23 */
  end: number;
}
```

Reply:
Resolved by updating comments for intellisense and `ITimeRange` fields.